### DISCUSSION TITLE
Always allow going back to fix something even in later stages

### DISCUSSION THREAD

**Discussion by jidanni** [community/community#113980] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

We are about to submit a Pull Request, here on GitHub.com!

We are already at the
> Comparing changes
Choose two branches to see what’s changed or to...

stage, when all the sudden we notice a slight grammar error.

We look around for a way to go back to fix it.

There is none. So we bite the bullet and push our browser's BACK button, and... all our work is lost.

Sure, you might say,
> You had your chance to fix it earlier in the sequence of events, and didn't take it.

But hey, it was showing me the diffs. It's not like the store was already closed when I went back for the exchange.

> Well serves you right for not doing the editing in Notepad, and then pasting the final draft into GitHub. That way you still have a copy just in case.

I guess I was too impressed with the GitHub editor's line numbers, etc.

> Well you can use this [very complicated method] to fix it.

Hey, I am just a junior programmer.

**Comment by Md-Inam** [comment]
better search it on youtube or gemini ai. dont foget to accept my asnwer if find helpful

**Reply by jidanni** [reply]
But that's the thing about needing big research. 

I think just a button to go back would be even better.

Just like all hotels have a fire alarm button somewhere.
